hoot-core/src/main/cpp/hoot/core/io/OsmXmlWriter.cpp

```diff
@@ -186,6 +201,17 @@ void OsmXmlWriter::_initWriter()
   _writer->writeAttribute("generator", HOOT_PACKAGE_NAME);
 }
 
+uint64_t OsmXmlWriter::getPos()
+{
+  return _fp->pos();
+}
+
+void OsmXmlWriter::flush()
+{
+  //if (_fp && _fp->isOpen())
+  //  _fp->
+}
+
```

Comment:
Is the flushing code not necessary or just not implemented yet?

Reply:
Good catch! This is dead code - I changed directions, and instead of implementing a flush(), opened the underlying file as un-buffered I/O. I'll just remove this code...